src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,6 +1,19 @@
-// Don't forget to import the React library
+import React from 'react';
 
-// Create a `TodoInfo` component accepting a `todo` object and use it in the
-// list to render `title`, `completed` status and `User`
+import './TodoInfo.scss';
 
-// Add a default export statement for TodoInfo component to use it in the other files
+type Props = {
```

Comment:
```suggestion
type Props = Todo;
```
simplify your life :)